src/components/Table/GridTable.tsx

```diff
@@ -341,10 +349,9 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
       // We only pass sortState to header rows, b/c non-headers rows shouldn't have to re-render on sorting
       // changes, and so by not passing the sortProps, it means the data rows' React.memo will still cache them.
       const sortProps = row.kind === "header" ? { sortOn, sortState, setSortKey } : { sortOn };
-      const RowComponent = observeRows ? ObservedGridRow : MemoizedGridRow;
```

Comment:
**question:** Did we get rid of `MemoizedGridRow`? I don't think its being used still is it?

Reply:
Ah damn; @bdow / @JonnCharpentier it looks like using `ObservedGridRow` means that (even before Jonn's PR) we would just never use `MemoizedGridRow`.

I think this was probably a bug/mistake in the old code, b/c the idea was less of "choose either observed or memoized" (which is what it was doing) but the intention was "observed would be layered on top of memoized".

@JonnCharpentier I wonder if we can try:

```
const MemoizedGridRow = React.memo(observer(GridRow), (one, two) => {
  const { row: row1, ...others1 } = one;
  const { row: row2, ...others2 } = two;
  return shallowEqual(row1, row2) && shallowEqual(others1, others2);
}) as typeof GridRow;
```

And then make line 354 `<MemoizedGridRow ...`.

I believe this will achieve both goals, of memoizing the component based on the `row` prop, but also wrapping it in an observer (now using the out-of-the-box `observer` function instead of the previous hand-written `ObservedGridRow` which was just doing the same thing).